data/geany.glade

```diff
@@ -3479,6 +3496,7 @@
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
                                         <property name="tooltip_text" translatable="yes">The amount of characters which are necessary to show the symbol autocompletion list</property>
+                                        <property name="invisible_char">●</property>
```

Comment:
Spurious change.  This probably comes form using a different version of Glade than the one we recommend?